src/Components/Main/Comment.js

```diff
@@ -0,0 +1,99 @@
+/*eslint-disable*/
+
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import CommentList from './CommentList.js';
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      newReply: '',
+      replies: [],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch('http://localhost:3000/data/Jungwoo/commentData.json', {
+      method: 'GET', // GET method는 기본값이라서 생략이 가능합니다.
+    }) // 예시코드에서는 이해를 돕기 위해 명시적으로 기입해뒀습니다.
```

Comment:
```suggestion
    fetch('/data/Jungwoo/commentData.json')
```
복사해온 코드라도 조금이라도 고쳐서 자기 것으로 만드셔야합니다!